crud/common/utils.lua

```diff
@@ -96,8 +96,26 @@ function utils.format_replicaset_error(replicaset_uuid, msg, ...)
     )
 end
 
-function utils.get_space(space_name, replicasets)
-    local replicaset = select(2, next(replicasets))
+function utils.get_space(space_name, vshard_router)
+    local replicasets, replicaset
+    local deadline = fiber.clock() + const.GET_SPACE_TIMEOUT
+    while (
+        -- Break if the deadline condition is exceeded.
+        -- Handling for deadline errors are below in the code.
+        fiber.clock() < deadline
+    ) do
+        -- Try to get master with timeout.
+        fiber.yield()
+        replicasets = vshard_router:routeall()
+        replicaset = select(2, next(replicasets))
+        if replicaset ~= nil and
+           replicaset.master ~= nil and
+           replicaset.master.conn.error == nil then
+            -- Break if there is master in replicaset and
```

Comment:
I think this comment is excessive, but up to you

Reply:
Done, deleted